wiki/Tournaments/SS_Tournament/2018/en.md

```diff
@@ -0,0 +1,205 @@
+| Round of 16 and Losers round 1      | 2018-02-16/2018-02-18     |
+| Quarter Finals and Losers round 2    | 2018-02-23/2018-02-25     |
+| Semi Finals and Losers round 3       | 2018-03-02/2018-03-04     |
+| Finals and Losers round 4 & 5       | 2018-03-09-2018-03-11     |
+| Grand Finals                         | 2018-03-16/2018-03-18     |
+
+## Rules
+
+### General Info
+
+* Teams may consist of 4 Members. 
+* There is no rank range the higher the rank (1 being highest) the more likely you are to avoid being cut off from a rank cut off of overflow sign ups
+* Seeding will take into account all Players PP. '(formula being <Member 1 PP> + <Member 2 PP> + …. / <# of players on team>)'
+* The tournament will be held in 3v3 double-elimination format with Groups having one top high mid low and bottom seed selected from the tournament drawings client.
+Brackets will then be made with the top placing team in group A vs the second top placing seed in group H, the second seed of group A facing the top placing team of group A, Group B will match up G and so fourth.
+* The matches will be played using Score V2 and TeamVS.
+* Map pools will be released The Monday before the stage starts.
+* Teams must reschedule 12 Hours in Advance to the start of the current stage. If for whatever reason a team needs to reschedule during the stage, it will be done on a case by case basis with the opposing team being prioritized if a reschedule cannot be arranged.
+* Staff is not allowed to play excluding commentators and streamers.
+* Group Stage Matches will be best of 5
+* Round of 16 will be best of 7
+* Quarter Finals and Semi Finals will be best of 9
+* Finals will be best of 11
+* Grand Finals will be best of 13
+* In the case of a tie in group stage, team advancement will be based off a team's '[map wins - map losses = Points]' where higher points advance. If the score is equal, the score of Team A Vs Team B will be taken into account, i.e. Team A and Team B both have 6 wins and 6 losses, but Team A beat Team B 3-2 Team A will advance.If there is still a tie it will be average score difference gathered by the formula '[Team A's scores of all played maps - All opposing teams scores / maps played]'.If there is still a tie after this, it will be based off seeding.
+
+### Match Rules
+
+1. The designated referee will have the lobby open using the format SST: (blue team) vs. (red team)
+1. The referee will message the team captains or anyone online from the team.
+1. Teams will join here. Once all 6 players are in the referee will ask for their warm ups. Order in which they are played is irrelevant. (this step is skipped for group stages)
+1. Both teams will '!roll', the higher number rolled will be the team who will ban first 
+1. Both teams will alternate banning maps until both teams have banned out 2 maps
+1. The team that lost the roll is the team that picks first and it it alternates from there. 
```

Comment:
WTF is these numbric lists???

Reply:
will fix when home

Reply:
using `1.` for all of the number list is okay to do